app/controllers/katello/api/v2/flatpak_remotes_controller.rb

```diff
@@ -85,5 +85,9 @@ def default_sort
     def flatpak_remote_params
       params.require(:flatpak_remote).permit(:name, :url, :description, :organization_id, :username, :token)
     end
+
+    def rejected_autocomplete_items
+      ['seeded', 'organization_id']
```

Comment:
I am thinking we can drop the seeded field altogether..Since the seeding proposal didn't go anywhere, this field isn't exactly needed.

Reply:
This was added as part of https://github.com/Katello/katello/blob/master/db/migrate/20241030181402_create_katello_flatpak_tables.rb and can be dropped in another migration since we don't need it..

Reply:
NVM..Discussed offline and we can keep the field to support future enhancement around seeding when we work on it..